portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACDataControlAndPrivacy.path

```diff
@@ -0,0 +1,46 @@
+<html>
+<head>
+<title>ACDataControlAndPrivacy</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">ACDataControlAndPrivacy</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>RESQUEST_LOG</td>
+	<td>//h4[contains(.,'Request Log')]/../..//a[contains(@class,'button') and contains(.,'Manage')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CREATE_REQUEST</td>
+	<td>//*[contains(@class, 'button-root btn btn-primary nav-btn') and contains(.,'Create Request')]</td>
```

Comment:
Hi @AfonsoHSantos. Since you're using `contains` then it's better to just use a part of that **class** so you can reduce the chance that a code change will break your test.

Suggestion: `//*[contains(@class, 'btn-primary') and contains(.,'Create Request')]`

Reply:
fixed